src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search-big-one {
+  margin: 20px 0;
+}
+
+.big-search-input {
```

Comment:
Text color of your inputs is a bit wrong from what's shown on figma, you should check it again